types/node/index.d.ts

```diff
@@ -40,6 +40,7 @@
 //                 NodeJS Contributors <https://github.com/NodeJS>
 //                 Linus Unnebäck <https://github.com/LinusU>
 //                 wafuwafu13 <https://github.com/wafuwafu13>
+//                 Dave Cardwell <https://github.com/davecardwell>
```

Comment:
fact check: you will be notified on any future updates to a code as part of code reviews and your input on changes will be welcome (a.k.a. you'll become owner of the implementation here)
is that your intent?

Reply:
@peterblazejewicz Yes that’s no problem, I have a this on a few other type definitions already so am familiar with the process.